config/config/sc-config.yaml

```diff
@@ -265,6 +265,8 @@ prometheus:
           memory: 100Mi
   ## the percentage of use the resources are set to hit in 3 days
   predictLinearLimit: 66
+  s3BucketPercentLimit: 80
+  s3BucketMaxSizeLimit: 1000
```

Comment:
is this in KiB or MiB?

Reply:
daily-checks/s3 total usage

Reply:
maybe add a comment that you are using GB or add it in the name. e.g. `s3BucketMaxSizeLimitGB`